frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -193,16 +192,29 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps> {
     );
   }
 
+  renderPanelHeader = (hasDisabledLayers: boolean) =>
+    hasDisabledLayers ? (
+      <span>
+        Color Layers
+        <Tooltip title="Not all layers are currently visible.">
+          <Icon type="exclamation-circle-o" style={{ marginLeft: 16, color: "coral" }} />
+        </Tooltip>
+      </span>
+    ) : (
+      "Color Layers"
+    );
+
   render() {
     const { layers } = this.props.datasetConfiguration;
     const colorSettings = Object.entries(layers).map((entry, index) =>
       // $FlowFixMe Object.entries returns mixed for Flow
       this.getColorSettings(entry, index, index === _.size(layers) - 1),
     );
-
+    const hasDisabledLayers =
+      Object.keys(layers).find(layerName => layers[layerName].isDisabled) != null;
```

Comment:
Maybe add `|| layers[layerName].alpha === 0` to the condition, this way the warning message "Not all layers are currently visible." would be even more appropriate :)